### DISCUSSION TITLE
Using without installing as a plugin

### DISCUSSION THREAD

**Discussion by andresespinosapc** [Akryum/v-tooltip#902] [Q&A]
Is there a way of using the Tooltip component without installing a Vue plugin to not use bundle size for the whole app?

I managed to do it importing it as `import { Tooltip } from 'floating-vue'`, but I don't know how to handle the themes.

**Comment by Akryum** [comment]
floating-vue is not tree-shakable and it wouldn't make sense as 90% of the code is the core component used for both tooltips and dropdowns.

**Reply by andresespinosapc** [reply]
But the package would only be loaded when needed, even if it's the whole package. At least when using lazy components or lazy routes that'd be the case.

**Reply by Akryum** [reply]
Anyway you can import the plugin and use the `options` property as explained in the docs:
https://floating-vue.starpad.dev/guide/config.html
